src/prefect/backend/flow.py

```diff
@@ -0,0 +1,284 @@
+from typing import Union, List, Dict, Tuple, Any
+
+import prefect
+from prefect.serialization.flow import FlowSchema
+from prefect.utilities.graphql import with_args, EnumValue
+from prefect.utilities.logging import get_logger
+
+
+logger = get_logger("backend.flow")
+
+
+class FlowMetadata:
+    """
+
+    Attributes:
+        flow_id: The uuid of the flow
+        flow: A deserialized copy of the flow. This is not loaded from storage, so tasks
+            will not be runnable but the DAG can be explored.
+        settings: A dict of flow settings
+        run_config: A dict representation of the flow's run configuration
+        serialized_flow: A serialized copy of the flow
+        archived: A bool indicating if this flow is archived or not
+        project_name: The name of the project the flow is registered to
+        core_version: The core version that was used to register the flow
+        storage: The deserialized Storage object used to store this flow
+        name: The name of the flow
+
+
+    TODO: Consider changing this class name. We don't want to to overlap with
```

Comment:
+1, this naming could be confusing

Reply:
Do you have a better name? :D

Reply:
Not yet! But I'm deliberately commenting and then not thinking about it, with hopes a better name will arrive in my brain later 😄